src/gt4py/next/errors/__init__.py

```diff
@@ -0,0 +1,41 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2023, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from . import (  # noqa: module needs to be loaded for pretty printing of uncaught exceptions.
```

Comment:
I would add docstrings to all the submodules, or at  least a docstring here for the whole 'errors" subpackage.